db/schema.rb

```diff
@@ -10,14 +10,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_04_14_073843) do
+ActiveRecord::Schema[7.0].define(version: 2022_04_15_125558) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
   enable_extension "plpgsql"
 
   # Custom types defined in this database.
   # Note that some types may not work with other database engines. Be careful if changing database.
-  create_enum "challenge_status", ["draft", "moderation", "ready", "canceled"]
+  create_enum "challenge_status", ["draft", "moderation", "pending", "registration", "live", "complete", "canceled"]
```

Comment:
please, run:
rake db:drop
rake db:create
rake db:migrate

and commit schema chamges again

Reply:
Thanks. I've corrected the schema.